### DISCUSSION TITLE
Add check for update Button

### DISCUSSION THREAD

**Discussion by cfragkos** [coollabsio/coolify#1652] [Feature Requests]
Add check for update Button

**Comment by andrasbacsai** [comment]
For which resource? Services?

**Reply by cfragkos** [reply]
For coolify

**Reply by Wulfheart** [reply]
@andrasbacsai I think it would be really cool to have coolify in the background if there are newer versions of the image and show a button that user can choose to install the latest version. Optionally even with a popup that warns the user if he REALLY, REALLY wants to update as he should have a backup of it.
